lms/djangoapps/instructor/views/instructor_dashboard.py

```diff
@@ -787,6 +812,38 @@ def _section_open_response_assessment(request, course, openassessment_blocks, ac
     return section_data
 
 
+def _section_recap(request, course, recap_blocks, access):
+    """Provide data for the Recap dashboard section."""
+
+    course_key = course.id
+    recap_block = recap_blocks[0]
+    block, __ = get_module_by_usage_id(
+        request, unicode(course_key), unicode(recap_block.location),
+        disable_staff_debug_info=True, course=course
+    )
+    # Set up recap instructor dashboard fragment, pass data to the context
+    fragment = block.render('recap_blocks_listing_view', context={})
+    # Wrap the fragment and get all resources associated with this XBlock view
+    fragment = wrap_xblock(
+        'LmsRuntime', recap_block, 'recap_blocks_listing_view', fragment, None,
```

Comment:
Please, split all the arguments in a new line. Can you add the keyword name argument to each one? @andrey-canon